ts/features/fci/utils/__tests__/dates.test.ts

```diff
@@ -0,0 +1,34 @@
+import { daysBetweenDate } from "../dates";
+
+describe("Test dates utils", () => {
+  describe("Test daysBetweenDays", () => {
+    it("should return 0 if the two dates are the same", () => {
+      const date = new Date();
+      expect(daysBetweenDate(date, date)).toBe(0);
+    });
+    it("should return 1 if the two dates are consecutive", () => {
+      const date = new Date();
+      expect(
+        daysBetweenDate(new Date(), new Date(date.setDate(date.getDate() + 1)))
```

Comment:
Since you used `Math.abs` in `daysBetweenDate` did you test if this function gives the expected behavior also if you switch dates? E.g.

```ts
expect(
    daysBetweenDate(new Date(date.setDate(date.getDate() + 1)), new Date())
).toBe(1);
```

And shouldn't be useful to test this function against mock dates to test also some edge cases like year switch and 29th of February?

Reply:
The main purpose is to get the **number of days between two dates**. Also having two dates one in future and one in past the number of days returned by this function always are the days passed. Conceptually the timeline is embedded in the Date itself. It's a good observation though, I just want to check this function again on the edge case you mentioned. Thx.